kemeny_sourcemash.tex

```diff
@@ -43,15 +43,55 @@ \section{Project Overview}
 
 \section{User Instructions}
 
-While 
-
-\subsection{Production: www.sourcemash.com}
-
-When
-
-\subsection{Development: localhost}
-
-When
+Sourcemash is a web application that is hosted online at \url{www.sourcemash.com}. The current development release is available at \url{staging.sourcemash.com}, and the source code for Sourcemash is available at \url{https://github.com/sourcemash/Sourcemash}. Instructions for installation, execution, and testing in the local environment are included in the README and in �2.2 below.
```

Comment:
"Sourcemash is a web application that is hosted online at \url{www.sourcemash.com}" --> "Sourcemash is a web application, located at \url{www.sourcemash.com}"